designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware Config
+
+Exposing Firmware Config for VM.
+Firmware is placed within the scheduling section. It is related to CPU, and shoudn't be seperated from the rest of the CPU features. Because of that, it should be located next to other CPU features. Also, it affects performance, so it is related to "Resource Requirements".
+
+![VM page](img/Firmware-0-0.jpg)
+Firmware Config is placed in the Scheduling sction of the VM details tab.
+The default value of this field is "Default"
+
+![Config modal](img/Firmware-1-0.jpg)
+This modal is composed of three fields:
+- Bootload (Selection): BIOS (Default) and EFI
+- Serial (string): [The system-serial-number in SMBIOS]
+- UUID ( string): [UUID reported by the vmi bios. Defaults to a random generated UID]
+
+![Config modal](img/Firmware-1-1.jpg)
+Changing the Bootload to EFI
+
+![Config modal popovers](img/Firmware-1-2.jpg)
```

Comment:
assuming these would be accessed via a (i) icon?

Reply:
Yes, think it is an underline now. Will add a mention of it in the MD.

Reply:
I think it's only an underline in the details view but not in a modal with a form.